examples/src/main/java/com/mapbox/navigation/examples/core/FasterRouteActivity.kt

```diff
@@ -43,6 +43,10 @@ import kotlinx.android.synthetic.main.content_faster_route_layout.*
 import timber.log.Timber
 
 /**
+ * This example shows how to use the Navigation SDK's [FasterRouteObserver]
+ * and display the observer's faster routes that are returned if the
+ * device goes off of the original [DirectionsRoute].
+ *
```

Comment:
address @langsmith 's comment in the previous PR #2947 
https://github.com/mapbox/mapbox-navigation-android/pull/2947#issuecomment-626820820